File: src/main/java/org/dxworks/githubminer/dto/response/repository/pullrequests/PullRequestReview.java
package org.dxworks.githubminer.dto.response.repository.pullrequests;

import com.google.api.client.json.GenericJson;
import com.google.api.client.util.Key;
import lombok.Data;
import org.dxworks.githubminer.dto.commons.User;

@Data
public class PullRequestReview extends GenericJson {
    @Key
    private String state;
    @Key
    private User user;
    @Key
    private String body;
}

File: github-miner-core/src/main/kotlin/org/dxworks/githubminer/dto/response/repository/pullrequests/PullRequestReview.kt
package org.dxworks.githubminer.dto.response.repository.pullrequests

import com.google.api.client.json.GenericJson
import com.google.api.client.util.Key
import org.dxworks.githubminer.dto.commons.User

class PullRequestReview : GenericJson() {
    @Key
    var state: String? = null

    @Key
    var user: User? = null

    @Key
    var body: String? = null

    @Key("submitted_at")
    var submittedAt: String? = null
}
